.github/workflows/ci-tests.yml

```
@@ -0,0 +1,31 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: ci-tests
+
+# Trigger the workflow when:
+on:
+  # A push occurs to one of the matched branches.
+  push:
+    branches:
+      - master
+  # Or when a pull request event occurs for a pull request against one of the
+  # matched branches.
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  tests:
+    # NOTE: This name appears in GitHub's Checks API.
+    name: tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Go 1.14
+        uses: actions/setup-go@v1
+        with:
+          go-version: "1.14.x"
+      - name: Run tests
+        run: |
+          go build
+          go test ./...
```

Perhaps separate those two?

```suggestion
      - name: Build code
        run: |
          go build
      - name: Run tests
        run: |
          go test ./...
```